synapse/rest/admin/__init__.py

```diff
@@ -215,11 +215,12 @@ async def on_GET(
     ) -> Tuple[int, JsonDict]:
         await assert_requester_is_admin(self.auth, request)
 
-        purge_status = self.pagination_handler.get_purge_status(purge_id)
+        purge_status = await self.pagination_handler.get_delete_status(purge_id)
         if purge_status is None:
             raise NotFoundError("purge id '%s' not found" % purge_id)
 
-        return HTTPStatus.OK, purge_status.asdict()
+        # TODO active vs purging etc
+        return HTTPStatus.OK, purge_status.asdict(use_purge_history_format=True)
```

Comment:
Is this TODO still relevant?

Reply:
it is not relevant anymore, the munging from `purging` to `active` was the problem and is solved by `use_purge_history_format` (for now :) ).